build/ubuntu/Dockerfile.fast-intermediate

```diff
@@ -1,5 +1,5 @@
 ARG builder=quay.io/wire/ubuntu20-builder
-ARG deps=quay.io/wire/ubuntu-deps
+ARG deps=quay.io/wire/ubuntu20-deps:develop
```

Comment:
@akshaymankar is this correct?
In contrast `Dockerfile.intermediate` has `deps=quay.io/wire/ubuntu20-deps`

Reply:
I think both of them should have the `develop` tag, IIRC the `latest` tag is not published.

Reply:
Ok, changed tag to `develop`